content/utilities/active-players/plugin.js

```diff
@@ -0,0 +1,144 @@
+  return playerDisplayText;
+}
+
+const getPlayerElement = playerId => {
+  const playerNameElement = document.createElement('span');
+  playerNameElement.style["color"] = pg.getPlayerColor(playerId);
+  playerNameElement.innerText = `${getPlayerDisplayText(playerId)}`;
+  return playerNameElement;
+}
+
+const getTableRowDiv = (element1, text2) => {
+  const element2 = document.createElement('span');
+  element2.innerText = String.fromCharCode(160) + `${text2}`;
+  const playerRowDiv = document.createElement('div');
+  playerRowDiv.append(element1);
+  playerRowDiv.append(element2);
+  return playerRowDiv;
+};
+
+class ActivePlayers {
+  constructor() {
+    this.finding = false;
+    this.tableRows = document.createElement("div");
+    if (DEV_MODE) this.tableRows.style.background = "#FF0000";
+    console.log(`Initialised ${PLUGIN_NAME} plugin:`);
+    console.dir(this);
+  }
+
+  async listActivePlayers() {
+    // Only proceed if not already finding
+    if (this.finding) return;
+    this.finding = true;
+    console.log("Listing Active Players");
+
+    // Remove existing table rows
+    while (this.tableRows.firstChild) {
+      this.tableRows.removeChild(this.tableRows.lastChild);
+    }
+
+    // Get moves (arrivals) data from The Graph
+    const playerQueryResults = await getGraphQLData(playerQuery);
+
+    // Count moves for each playerId
+    const activePlayerMap = {};
+    for (let arr of playerQueryResults.data.arrivals) {
+      const playerId = arr.player.id;
+      if (activePlayerMap[playerId]) {
+        activePlayerMap[playerId] += 1;
+      } else {
+        activePlayerMap[playerId] = 1;
+      }
+    }
+
+    // Turn this map into an array, sort by decreasing count
+    let activePlayers = [];
+    for (let playerId of Object.keys(activePlayerMap)) {
+      activePlayers.push([playerId, activePlayerMap[playerId]]);
+    }
+    activePlayers = activePlayers.sort((a, b) => b[1] - a[1]);
+
+    // Table header
+    this.tableRows.append(getTableRowDiv("Player Name" + String.fromCharCode(160).repeat(8), "Moves"));
+    
+    // Table content
+    for (let arr of activePlayers) {
+      const playerId = arr[0];
+      const count = arr[1];
+      this.tableRows.append(getTableRowDiv(getPlayerElement(playerId), count));
+    }
+    this.finding = false;
+  }
+
+  async render(container) {
+    container.style.width = "220px";
+    const searchButton = document.createElement('button');
+    searchButton.innerText = 'List Active Players!';
+    searchButton.onclick = this.listActivePlayers.bind(this);
+    const searchRow = document.createElement('div');
+    searchRow.style.margin = '15px 0px 10px 0px';
+    searchRow.appendChild(searchButton);
+    container.appendChild(searchRow);
+    container.appendChild(this.tableRows);
+  }
+
+  destroy() {
+  }
+}
+
+export default ActivePlayers;
```

Comment:
```suggestion
export default ActivePlayers;

```
nitpick github nags about newline.
![Screenshot 2022-03-31 at 14 17 39](https://user-images.githubusercontent.com/155505/161052913-d989cca0-cc5e-4691-a7e2-f49c24786438.png)